to-do-list/src/components/TaskLists.js

```diff
@@ -0,0 +1,31 @@
+import Task from './Task'
+
+function TaskList(props) {
+
+    const { tasks, status, completeTask, deleteTask, curDate } = props
+
+
```

Comment:
lots of empty space in this file. Consider using a code formatter like Prettier to clean this up automatically and make it easier to read!